ansible/README.md

```diff
@@ -25,7 +25,7 @@ brew install python
 pip install ansible==2.0.2.0
 
 cd ansible
-ansible-playbook setup.yml [-e docker_machine_name=whisk]
+ansible-playbook -i environments/mac setup.yml [-e docker_machine_name=whisk]
```

Comment:
no, it's for setup.yml too. `setup.yml` requires a specific environment for now, cause it needs the ip of db group which is different in mac and local. Once we can unite mac and local env, we still have to input the env here.